config/webpack.test.js

```diff
@@ -27,9 +27,18 @@ module.exports = {
         use: 'ts-loader'
       },
       {
-        test: /\.js$/,
-        use: ['source-map-loader'],
+        test: /\.[tj]sx?$/,
```

Comment:
Is this just a standard configuration now or are we really doing jsx and tsx stuff? In contrast in other locations we're still using just straight `.ts` file extensions.

Reply:
It's just a standard thing, we have no need for jsx/tsx so it isn't that big of a deal I don't think.